packages/@aws-cdk/aws-scheduler-alpha/test/group.test.ts

```diff
@@ -101,18 +102,18 @@ describe('Schedule Group', () => {
     const schedule1 = new Schedule(stack, 'MyScheduleDummy1', {
       schedule: expr,
       group: group,
-      target: new targets.LambdaInvoke({
+      target: new targets.LambdaInvoke(func, {
```

Comment:
Putting target resource before properties. I think it makes more sense than other way around.

Reply:
Agreed :-)